src/web-schemas.ts

```diff
@@ -156,7 +156,22 @@ export enum WebAppMsgType {
   RED_ENVELOPES            = 2001,
   READER_TYPE              = 100001,
 }
-
+/**
+ * WebMessageSendStatus from webwx-app
+ *  @see https://github.com/wechaty/webwx-app-tracker/blob/a12c78fb8bd7186c0f3bb0e18dd611151e6b8aac/formatted/webwxApp.js#L7520-L7524
+ *
+ *  //msg send status
+ *    MSG_SEND_STATUS_READY: 0
+ *    MSG_SEND_STATUS_SENDING: 1
+ *    MSG_SEND_STATUS_SUCC: 2
+ *    MSG_SEND_STATUS_FAIL: 5
+ */
+export enum WebMessageSendStatus{
```

Comment:
Could you please rename `WebMessageSendStatus` to `MsgSendStatus` so that it will follow the webwx-app naming convention?

Reply:
Sure.